src/screens/Member/styles.ts

```diff
@@ -0,0 +1,111 @@
+import {StyleSheet} from "react-native";
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1, justifyContent: 'center'
+    },
+    textInput: {
+        height: 50,
+        borderColor: '#dbdbdb',
```

Comment:
it's more recommended to export colors  and reuse them . in the project this file [theme.ts](https://github.com/Doha26/Instagram-clone/blob/master/src/utils/theme.ts) host some reusable colors.